test/index.js

```diff
@@ -143,4 +145,22 @@ describe('mem-fs', function () {
       });
     });
   });
+
+  describe('type-declaration', function() {
```

Comment:
Let's not run this through mocha. Instead, how about we just call `tsc -p ./folder/...` from the `npm test` command.